gulpfile.js

```diff
@@ -1,42 +1,36 @@
-var gulp         = require('gulp');
-var sass         = require('gulp-sass');
-var browserSync  = require('browser-sync').create();
-var useref       = require('gulp-useref');
-var gulpIf       = require('gulp-if');
-var uglify       = require('gulp-uglify');
-var cssnano      = require('gulp-cssnano');
-var imagemin     = require('gulp-imagemin');
-var cache        = require('gulp-cache');
-var del          = require('del');
-var runSequence = require('run-sequence');
+const gulp        = require('gulp');
+const pl          = require('gulp-load-plugins')();
+const browserSync = require('browser-sync').create();
+const runSequence = require('run-sequence');
+const del          = require('del');
```

Comment:
This "=" being out of place is kiling me, I don't want to pollute the commit, so I'll wait until you've seen it, but, I'll definitely want to update that as well.